content/definitions/czym-jest-html.md

```diff
@@ -2,5 +2,11 @@
 question: Czym jest HTML?
 ---
 
-Jest to *język*, rozumiany przez przeglądarki internetowe,
-umożliwiający wyświetlanie treści w odpowiednio zorganizowany sposób.
\ No newline at end of file
+`HyperText Markup Language` - jest to język, rozumiany przez przeglądarki internetowe,
+umożliwiający wyświetlanie treści w odpowiednio zorganizowany sposób.
+
+Tworzymy *szablon* strony z odpowiednich elementów (np `div`, `p`, `header`, `section`) tak, aby całość miała 
+logiczny sens. Możemy to traktować jak strona tytułowa gazecie, pozbawiona jakichkolwiek wizualnych dodatków.
+
+Najnowsza oficjalna wersja, to `HTML 5`, która wprowadziła sporo nowości. Jednak minęło już kilka lat od wypuszczenia 
```

Comment:
that comma is not needed here I guess `Najnowsza oficjalna wersja to...`

Reply:
Yeah, I get your point, but what I meant to say is that we don't need anymore to underline that we are using HTML5. Few years before it wasn't so obvious that you are using semantic tags etc, but now it's something natural so saying that you are writing in HTML 5 is something like.. 'I have a flat TV in my house' - ok good for you, but all TVs are now flat xD

Reply:
will try to refactor it tho

Reply:
Ok, I shortened it a bit, you can check it out br0000 ✋🏻